pyproject.toml

```diff
@@ -43,9 +43,10 @@ poethepoet               = "^0.16.4"
 [tool.poe.tasks]
 test       = "pytest tests -m \"(not slow) and (not cloud)\""
 doctest    = "pytest --doctest-modules docs"
-format     = "black --exclude \".*\\.pyi\" --check systemlink examples tests"
-lint       = "flake8 systemlink examples tests"
-types      = "mypy --config-file mypy.ini systemlink examples/tag tests"
+check      = "black --exclude \".*\\.pyi\" --check nisystemlink examples tests"
```

Comment:
@mure - I changed "format" to "check" and updated the workflow to call this one instead, and then added a new "format" task that actually performs the format.

Reply:
Makes sense - thanks